cross-project-tests/lit.cfg.py

```diff
@@ -37,7 +37,10 @@
 tools = [
     ToolSubst(
         "%test_debuginfo",
-        command=os.path.join(
+        command="LLVM_LIBS_DIR="
+        + config.llvm_libs_dir
+        + " "
+        + os.path.join(
```

Comment:
why don't we pass in the right python executable and the full path to the LLDB python packages here, then there doesn't need to be any guesswork in the other two files.